### PR TITLE
Patient hibernate model

### PR DESCRIPTION
**Pull request by mareo1997**

### CODE REVIEW

File: src/main/java/com/revature/model/Patient.java [hunk 1]
```diff
@@ -1,5 +1,238 @@
...
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="patient")
 public class Patient {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (quincy-roman) [inline src/main/java/com/revature/model/Patient.java:20]: Don't you need the @Column?

File: src/main/java/com/revature/model/Pokemon.java [hunk 2]
```diff
@@ -153,4 +153,5 @@ public String toString() {
 				+ ", ability=" + ability + ", category=" + category + "]";
 	}
 	
+
```
Comment (quincy-roman) [inline src/main/java/com/revature/model/Pokemon.java:156]: Typo?

File: src/main/java/com/revature/model/Patient.java (hunk 1 above)
Comment (mareo1997) [reply src/main/java/com/revature/model/Patient.java:20]: Forgot and fixed

File: src/main/java/com/revature/model/Pokemon.java (hunk 2 above)
Comment (mareo1997) [reply src/main/java/com/revature/model/Pokemon.java:156]: Where

**Pull request merged**